apps/sertra/components/GetModMan.tsx

```diff
@@ -0,0 +1,19 @@
+import styles from "./GetModMan.module.css";
+
+export const GetModMan = () => {
+  return (
+    <button
+      type="button"
+      className={styles.getModManButton}
+      onClick={(e) => {
+        e.preventDefault();
+        window.open(
```

Comment:
Okay so I think I've "fixed" it now. Let me know if it could have been done better. Also I'll be waiting for an answer from @gkurck for the `<a>` element thing.

Reply:
Yes, this should be `<a>`, as that's what it is - an external link. When you're dealing with either hyperlinks or external links, use `<a>`. When you are dealing with something toggle-able and/or programmable, such as submitting a form or toggling a menu, use `<button>`.

The "Join Server" -button should be `<a>` as well.